action.yml

```diff
@@ -16,7 +16,7 @@ inputs:
     default: ${{ github.token }}
     required: true
   enablement:
-    description: 'Should a Pages site be enabled for the repository if not so already? This will only work with user-to-server tokens.'
+    description: 'Should a Pages site be enabled for the repository if not so already? This will only work for user-to-server tokens, or server-to-server tokens with both the `pages: write` and `administration: write` permissions granted.'
```

Comment:
An acceptable alternative to getting this specific would just be to note that the Actions-provided `GITHUB_TOKEN` will not suffice for this. 🤔

Reply:
Maybe we should make it clear that you need to generate your own PAT for this, since `GITHUB_TOKEN` can't be created with those permissions.

Reply:
That's kind of what I was after with my initial change here but maybe my language was too indirect. Maybe even a combination of the initial change plus my suggestion from the comment? I'm waffling. 🤔

Reply:
Maybe 
```
This requires a personal access token  with both the `pages: write` and `administration: write` permissions granted, as GITHUB_TOKEN will not have sufficient privileges'
```
?

Reply:
That's not quite right either. 😅  It's complicated!

If we removed the "personal" from "personal access token", that could cover both the new fine-grained PATS + App server-to-server tokens, but it still misses the legacy PATs. Those have a different permissions model, so you would could get sufficient permissions either out-of-box for public repos or by adding the `repo` scope for private repos... I believe. 😅

Reply:
@yoannchaudet Any advice on the wording here? 👂🏻 🙏🏻